Makefile

```diff
@@ -77,11 +77,14 @@ ifndef HAS_DEP
 endif
 	dep version
 
-test: clean-last-testrun test-unit test-cli
+test: clean-last-testrun test-unit test-integration test-cli
 
-test-unit: build
+test-unit:
```

Comment:
Oh yea, re: the test unit Brigade Check failure... I guess previously we had called `build` which set up/populated the `bin` dir in a certain way that it appears the tests still expect.

I do like that this target doesn't invoke `build` anymore, though.  Maybe we add `make build` before the `test-unit` invocation to the `testUnit` function in the `brigade.js` file instead?

Reply:
I think that I need to fix those tests in particular that rely on copying binaries from the bin to either not rely on that, or move them over to be this new integration tests type. In a follow up PR though since this is getting big.

Reply:
https://github.com/deislabs/porter/issues/397